### ISSUE TITLE
[Bug]: Taint issue

### ISSUE THREAD

**Issue by sthdevilish** [ascott18/TellMeWhen#1991]
### WoW Version

Retail

### TellMeWhen Version

10.0.1

### Describe the bug

Occured randomly after starting combat, can't press most of my skills from action bar:

### Export Strings

```text
1x [ADDON_ACTION_BLOCKED] AddOn 'TellMeWhen_Options' tried to call the protected function 'UIParentBottomManagedFrameContainer:SetPoint()'.
[string "@Interface/AddOns/!BugGrabber/BugGrabber.lua"]:480: in function <Interface/AddOns/!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function `SetPoint'
[string "@Interface/FrameXML/UIParent.lua"]:3078: in function `UIParentManageFramePositions'
[string "@Interface/FrameXML/UIParent.lua"]:2561: in function <Interface/FrameXML/UIParent.lua:2548>
[string "=[C]"]: in function `SetAttribute'
[string "@Interface/FrameXML/UIParent.lua"]:3104: in function `UIParent_ManageFramePositions'
[string "@Interface/AddOns/Blizzard_ObjectiveTracker/Blizzard_ObjectiveTracker.lua"]:1470: in function <...zzard_ObjectiveTracker/Blizzard_ObjectiveTracker.lua:1365>
[string "=[C]"]: in function `ObjectiveTracker_Update'
[string "@Interface/AddOns/Blizzard_ObjectiveTracker/Blizzard_ObjectiveTracker.lua"]:878: in function <...zzard_ObjectiveTracker/Blizzard_ObjectiveTracker.lua:876>
```

**Comment by ascott18** [comment]
Unfortunately, Blizzard's new interface in 10.0 is severely broken and will randomly blame pretty much any addon for things like this. Until they make their error messages better (something Ghostcrawler promised back in Cataclysm...), I'm not going to waste time trying to chase down these kinds of things if they aren't consistently reproducible.

**Comment by sthdevilish** [comment]
Just as I thought as I encountered similiar errors blocking me from actionbar usage with few different addons. Hopefully it will be fixed in 10.0.2, but I can't test it on beta due to many addons being broken again :( This is going be awful game launch for DF because of this.

**Comment by ascott18** [comment]
Its worth noting that action bar addons don't seem to have this issue. Bartender4, Dominos, and such. I really wanted to try to revert to the default ones in 10.0 but this issue made that impossible.

**Comment by sthdevilish** [comment]
Let's hope they will fix it in 10.0.2, for now I'm using NoTaint addon (doesn't always work but I want to believe it helps to reduce errors). Just a shame they didn't test/fix 10.0.0 enough before releasing, DF launch is going to be a disaster with half of the addons not working again due to changes they made after releasing 10.0 (but only on beta, so I've heard). After 18 years Blizzard still believing people would play their game without addons...

**Comment by ascott18** [comment]
seems like maybe this is less of a dumpsterfire from blizzard (or maybe i just don't notice since I don't use the default action bars), so going to close this out for now.
